lib/tasks/migrations/plans/load_dc_benefit_market_catalog.rake

```diff
@@ -40,6 +40,26 @@ namespace :load do
         product_class.by_product_package(product_package).collect { |prod| prod.create_copy_for_embedding }
       end
 
+      if kind == :aca_shop
+        puts "Creating eligibilities......"
+
+        result = Operations::Eligible::CreateCatalogEligibility.new.call(
+          {
+            subject: benefit_market_catalog.to_global_id,
+            eligibility_feature: "aca_shop_osse_eligibility",
+            effective_date: benefit_market_catalog.application_period.min.to_date,
+            domain_model: "AcaEntities::BenefitSponsors::BenefitSponsorships::BenefitSponsorship"
+          }
+        )
+
+        if result.success?
+          p "Success: created eligibility for #{current_period_bc.start_on.year} benefit coverage period"
```

Comment:
here we don't get **current_period_bc** value. We do get benefit_market_catalog

Reply:
+1